src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -13,6 +15,40 @@ import "./IPausable.sol";
  */
 
 interface IEigenPodManager is IPausable {
+    /**
+     * Struct type used to specify an existing queued withdrawal. Rather than storing the entire struct, only a hash is stored.
+     * In functions that operate on existing queued withdrawals -- e.g. `startQueuedWithdrawalWaitingPeriod` or `completeQueuedWithdrawal`,
+     * the data is resubmitted and the hash of the submitted data is computed by `calculateWithdrawalRoot` and checked against the
+     * stored hash in order to confirm the integrity of the submitted data.
+     */
+    struct BeaconChainQueuedWithdrawal {
+        // @notice Number of "beacon chain ETH" virtual shares in the withdrawal.
+        uint256 shares;
+        // @notice Owner of the EigenPod who initiated the withdrawal.
+        address podOwner;
+        // @notice Nonce of the podOwner when the withdrawal was queued. Used to help ensure uniqueness of the hash of the withdrawal.
+        uint96 nonce;
+        // @notice Block number at which the withdrawal was initiated.
+        uint32 withdrawalStartBlock;
+        // @notice The operator to which the podOwner was delegated in EigenLayer when the withdrawal was created.
+        address delegatedAddress;
+    }
+
+    /**
+     * @notice Struct used to track a pod owner's "undelegation limbo" status and associated variables.
+     * @dev Undelegation limbo is a mode which a staker can enter into, in which they remove their virtual "beacon chain ETH shares" from EigenLayer's delegation
+     * system but do not necessarily withdraw the associated ETH from EigenLayer itself. This mode allows users who have restaked native ETH a route via
+     * which they can undelegate from an operator without needing to exit any of their validators from the Consensus Layer.
+     */
+    struct UndelegationLimboStatus {
+        // @notice Whether or not the pod owner is in the "undelegation limbo" mode.
+        bool undelegationLimboActive;
```

Comment:
the `undelegationLimbo` in the names of these are redundant imo

Reply:
agreed, holdover from before I combined into a struct, will revise names

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/135/commits/b16368ba48b11de89bc5291ff0b644a9d1fe1019